Observer/OfflinePaymentObserver.php

```diff
@@ -39,32 +59,39 @@ public function execute(Observer $observer)
     {
         $order   = $observer->getEvent()->getOrder();
         $payment = $order->getPayment();
-
-        if ($payment->getAdditionalData('payment_type') !== 'bill_payment_tesco_lotus') {
-            return $this;
+        $emailData     = new \Magento\Framework\DataObject();
+        $charge_id = $payment->getAdditionalInformation('charge_id');
+        $charge = $this->charge->find($charge_id);
+        switch($payment->getAdditionalInformation('payment_type')) {
+            case 'bill_payment_tesco_lotus':
+                // make sure timezone is Thailand.
+                date_default_timezone_set("Asia/Bangkok");
+                $barcodeHtml   = $this->_helper->convertTescoSVGCodeToHTML($payment->getAdditionalInformation('barcode'));
+                $emailTemplate = 'send_email_tesco_template';
+                $validUntil    = date("d-m-Y H:i:s" , strtotime($charge->expires_at));
+                break;
+            case 'paynow':
+                // make sure timezone is Singapore.
+                date_default_timezone_set("Asia/Singapore");
+                $barcodeHtml   = "<img src= '".$charge->source['scannable_code']['image']['download_uri']."'/>";
+                $emailTemplate = 'send_email_paynow_template';
+                $emailData->setData(['banksUrl' => $this->_assetRepo->getUrl('Omise_Payment::images/supportedbanks.png')]);
```

Comment:
> `supportedbanks.png`

There's nothing in the name or its placement in the filesystem that indicates that this file is specific to PayNow.

Reply:
done